src/libs/PolicyUtils.ts

```diff
@@ -132,15 +132,15 @@ function getNumericValue(value: number | string, toLocaleDigit: (arg: string) =>
 /**
  * Retrieves the distance custom unit object for the given policy
  */
-function getCustomUnit(policy: OnyxEntry<Policy>): CustomUnit | undefined {
+function getDistanceRateCustomUnit(policy: OnyxEntry<Policy>): CustomUnit | undefined {
```

Comment:
```suggestion
function getDistanceCustomUnit(policy: OnyxEntry<Policy>): CustomUnit | undefined {
```

Reply:
NAB. But the custom unit is for the distance not the distance rate. As the custom unit is km, miles. Where the "unit" for the rate is the currency. (Also CONST.CUSTOM_UNITS.NAME_DISTANCE is "Distance")